src/coreclr/pal/src/exception/remote-unwind.cpp

```diff
@@ -2213,6 +2213,33 @@ get_proc_name(unw_addr_space_t as, unw_word_t addr, char *bufp, size_t buf_len,
     return -UNW_ENOINFO;
 }
 
+#ifdef FEATURE_USE_SYSTEM_LIBUNWIND
+
+// Function typedef for unw_get_proc_info_in_range
+typedef int (*unw_get_proc_info_in_range_fn)(unw_word_t start_ip, unw_word_t end_ip, unw_word_t eh_frame_table, unw_word_t eh_frame_table_len, unw_word_t exidx_frame_table, unw_word_t exidx_frame_table_len, unw_addr_space_t as, unw_word_t ip, unw_proc_info_t *pi, int need_unwind_info, void *arg);
+
+// Function to attempt to load unw_get_proc_info_in_range dynamically
+static unw_get_proc_info_in_range_fn
+get_unw_get_proc_info_in_range()
+{
+    #define STRINGIFY(x) #x
+    #define UPREFIX(fn) STRINGIFY(UNW_PASTE(UNW_PASTE(UNW_PASTE(_U, UNW_TARGET), _), fn))
+    #define ULPREFIX(fn) STRINGIFY(UNW_PASTE(UNW_PASTE(UNW_PASTE(_UL, UNW_TARGET), _), fn))
+
+    // Attempt to find the symbol in the global namespace using RTLD_DEFAULT
+    unw_get_proc_info_in_range_fn handle = (unw_get_proc_info_in_range_fn)dlsym(RTLD_DEFAULT, ULPREFIX(get_proc_info_in_range));
```

Comment:
`dlsym` is relative expensive operation. I would expect that the value should be cached to avoid leaving perf on the table.

The simplest way to do the caching is to wrap the initialization in pthread_once